documentation/modules/metrics/proc-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,62 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='cruise-control-monitoring-{context}']
+= Presenting Cruise Control metrics in Grafana
```

Comment:
All the _Presenting ... metrics in Grafana_ seem weird. I do not think anyone would call it _presenting_. I think we should find some more natural verb for all the titles.

Reply:
All removed now